tigera-operator/v1.34.0/rockcraft.yaml

```diff
@@ -0,0 +1,119 @@
+name: calico-tigera-operator
+summary: Calico tigera-operator ROCK image.
+description: >
+  This rock is a drop in replacement for the
+  docker.io/tigera/operator:v1.34.0 image.
+version: v1.34.0
+license: Apache-2.0
+
+base: ubuntu@24.04
+build-base: ubuntu@24.04
+platforms:
+  amd64:
+
+entrypoint-service: tigera-operator
+services:
+  tigera-operator:
+    command: /usr/bin/operator [ ]
+    override: replace
+    startup: enabled
+
+parts:
+  build-deps:
+    plugin: nil
+    build-snaps:
+      - go/1.22/stable
+
+  tigera-operator:
+    after: [build-deps]
+    plugin: go
+    source-type: git
+    source: https://github.com/tigera/operator
+    source-tag: v1.34.0
+    source-depth: 1
+    override-build: |
```

Comment:
Can we keep the patches as `.patch` files and import/use them as we do with entrypoint scripts?